core/base/src/TROOT.cxx

```diff
@@ -623,7 +623,7 @@ ClassImp(TROOT);
 TROOT::TROOT() : TDirectory(),
      fLineIsProcessing(0), fVersion(0), fVersionInt(0), fVersionCode(0),
      fVersionDate(0), fVersionTime(0), fBuiltDate(0), fBuiltTime(0),
-     fTimer(0), fApplication(0), fInterpreter(0), fBatch(kTRUE), fEditHistograms(kTRUE),
+     fTimer(0), fApplication(0), fInterpreter(0), fBatch(kTRUE), fWeb(""), fIsWeb(0), fEditHistograms(kTRUE),
```

Comment:
`fIsWeb` should be initialized by `kFALSE`, not `0`. No need to initialize `fWeb` explicitly.